### PR TITLE
Release all api single command

### PR DESCRIPTION
**Pull request by siennamw**
Includes slight refactor to single-app deploy to make this aggregation task more DRY
Also fixed the deploy all task

### CODE REVIEW

**Comment by BriannaGreene** [comment]
can you update the README with how to deploy all API services at once?

File: README.md [hunk 1]
```diff
@@ -38,15 +38,17 @@ Deploy the "console" app to the "staging-uat" ECS cluster.
...
 
-## How to Deploy the Shipping Platform to Staging
-1. create a staging branch (Example: `staging/name-of-current-sprint`)
-2. merge in branch `GCT-314-get-shipping-working` (this branch is not in prod)
-3. Deploy shipping to staging-uat3
+#### Deploying All API Services Together
+Instead of deploying `consoleapi`, `web-api`, `worker`, and `scheduler` individually, you can instead use the following
+with the desired cluster name:
 ```bash
-jake deploy:app['staging-uat3','shipping-api']
```
Comment (BriannaGreene) [inline README.md]: Have to checked with SRE team to confirm that this method is outdated?

Comment (BriannaGreene) [reply README.md]: Does the SRE team have a different method for deploying shipping to staging then the standard method? If it's different then how we deploy other repos, we should update the README with their new method.

Comment (siennamw) [reply README.md]: @bearfact, @tom-con, @SealedSaint - can you answer @BriannaGreene's question about deploying `shipping-platform`?  This PR includes an update to the README that (in part) aims to remove deprecated instructions for `shipping-platform`.

**Comment by BriannaGreene** [comment]
I tested this new command and did not get a slack notification that scheduler was deployed. Is it shipping scheduler that gets deployed? I don't see any tasks running for shipping-scheduler on staging-uat.

File: jakelib/deploy.jake [hunk 2]
```diff
@@ -121,38 +121,92 @@ namespace('deploy', function () {
...
-    jake.exec(cmd, { printStdout: true }, function(){
+
+    jake.exec(cmds.join(' && '), { printStdout: true }, function(){
       jake.Task['ecs:restart'].execute(cluster_name, `${cluster_name}-${app_name}`);
       jake.Task['slack:deployment'].execute(cluster_name, app_name);
       complete();
     });
   });
 
+  const buildCmdString = (path, cluster_name, app_name) => {
```
Comment (SealedSaint) [inline jakelib/deploy.jake]: Declare above before use?

Comment (siennamw) [reply jakelib/deploy.jake]: Done!

File: README.md [hunk 3]
```diff
@@ -38,15 +38,17 @@ Deploy the "console" app to the "staging-uat" ECS cluster.
...
+jake deploy:apiall['staging-uat']
```
[the remark this answers is not in the document]
Comment (siennamw) [reply README.md]: Good call.  Done!

**Comment by siennamw** [comment]
@BriannaGreene The core script automatically excludes `scheduler` from deployments to staging environments because we aren't currently using `scheduler` in most staging environments.  It can still be released individually, and maybe we'll want to change this in the future.  This is covered in the README.

**Pull request merged**